locations/categories.py

```diff
@@ -232,6 +232,7 @@ class Fuel(Enum):
     OCTANE_98 = "fuel:octane_98"
     OCTANE_100 = "fuel:octane_100"
     # Formulas
+    E0 = "fuel:e0"
```

Comment:
I don't understand `E0`. If this were applicable, wouldn't one of the `OCTANE_` values be used instead?

Reply:
Good point. 

In case of Sheetz example, they have `e0` attribute, octane value is not specified, so I'm not sure what `OCTANE_` tag I should use in that case.

OSM wiki has `fuel:e0` tag

![image](https://github.com/alltheplaces/alltheplaces/assets/35237910/cac00205-a773-4993-9d6b-864a95483e56)

Reply:
`fuel:ethanol_free` us more used than `fuel:e0`, so if they are the same, usually I'd use that.

However when tags have usage this low, OSM could change quickly. So if we have a strong opinion, we can pick that.

Reply:
I looked up Sheetz ethanol free fuel and found some people online talking about how it has a 91 or 93 octane rating. Seemingly ethanol free fuel could have many different octane ratings.

The OCTANE_ tags are a bit silly because they don't explain the content of the fuel (which is a lot of different chemical compounds). So `fuel=` is overloaded to mean both an octane rating, and separately, the type of chemical composition of a fuel. Ideally a `fuel:wikidata=Q123` tag may exist, and the Wikidata item would provide the full composition of the fuel, link to safety data sheets, etc.

Reply:
> fuel:ethanol_free us more used than fuel:e0

Makes sense, changed in https://github.com/alltheplaces/alltheplaces/pull/5831/commits/ecc68c682013faa0c31e62ef350a99035e42babc.

Reply:
I can't really offer anything, I don't drive. But this conversation needs to happen in OSM land.